### PR TITLE
Allow specifying rent payer when creating NFT

### PR DESCRIPTION
**Pull request by vpontis**

### CODE REVIEW

File: program-tests/nft-create-v2.test.ts [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    const { nft_pubkey } = await createNftV2({ holder });
+    const nft = await program.account.nftAccount.fetch(nft_pubkey);
+
+    expect(nft.authority.toBase58()).toEqual(authority.toBase58());
+    expect(nft.holder.toBase58()).toEqual(holder.toBase58());
+    expect(nft.collection.toBase58()).toEqual(NULL_PUBKEY_STRING);
+    expect(nft.version).toEqual(1);
+  });
+
+  test("mints an NFT with a different payer", async () => {
```
Comment (sjchmiela) [inline program-tests/nft-create-v2.test.ts:43]: This test case is the de facto difference between `nft-create-v1.test.ts` and `nft-create-v2.test.ts`.

**Comment by sjchmiela** [comment]
The pull request is ready for your review!

File: program-tests/nft-create-v2.test.ts (hunk 1 above)
Comment (vpontis) [reply program-tests/nft-create-v2.test.ts:43]: Thanks! Looks good 👍

**Comment by vpontis** [comment]
@sjchmiela this is deployed to devnet + mainnet 👍

**Pull request merged**